.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,38 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: GCC multilib
+        run: |
+          sudo apt-get install gcc-multilib g++-multilib clang 
+      -
+        name: OSXCross for CGO Support
+        run: |
+          mkdir ../../osxcross
+          git clone https://github.com/plentico/osxcross-target.git ../../osxcross/target
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```

Comment:
`go.mod` depends on go-version 1.15, Should we have goreleaser use the same version?